pyanaconda/bootloader.py

```diff
@@ -2337,6 +2341,10 @@ def write_config_header(self, config):
                   "timeout={}\n"
                   "target=/boot\n")
         config.write(header.format(self.timeout))
+
+        if os.path.exists(util.getSysroot() + "/usr/sbin/new-kernel-pkg"):
+            flags.blscfg = False
+
         if not flags.blscfg:
```

Comment:
Is there a reason why the flag needs to be modified? Could you just change the condition to:
`if not flags.blscfg or os.path.exists(util.getSysroot() + "/usr/sbin/new-kernel-pkg"):`

I am afraid of the side effects.

Reply:
@poncovka thanks a lot for your feedback. The reason why was disabled was to prevent having the same check done multiple times, for example in the case of `ZIPL` class, the `flags.blscfg` are used in both `write_config_header()` and `write_config_images()`.

But you are correct about the possibly side effects, I'll re-spin the patch just adding a check for  `new-kernel-pkg` existence on every place where `flags.blscfg` is checked.